src/validation.cpp

```diff
@@ -1079,15 +1079,13 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
 
 } // anon namespace
 
-/** (try to) add transaction to memory pool with a specified acceptance time **/
-static MempoolAcceptResult AcceptToMemoryPoolWithTime(const CChainParams& chainparams, CTxMemPool& pool,
-                                                      CChainState& active_chainstate,
-                                                      const CTransactionRef &tx, int64_t nAcceptTime,
-                                                      bool bypass_limits, bool test_accept)
-                                                      EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+MempoolAcceptResult AcceptToMemoryPool(const CChainParams& chainparams, CTxMemPool& pool,
+                                       CChainState& active_chainstate, const CTransactionRef& tx,
+                                       int64_t accept_time, bool bypass_limits, bool test_accept)
+    EXCLUSIVE_LOCKS_REQUIRED(cs_main)
 {
     std::vector<COutPoint> coins_to_uncache;
-    auto args = MemPoolAccept::ATMPArgs::SingleAccept(chainparams, nAcceptTime, bypass_limits, coins_to_uncache, test_accept);
+    auto args = MemPoolAccept::ATMPArgs::SingleAccept(chainparams, accept_time, bypass_limits, coins_to_uncache, test_accept);
```

Comment:
can this be `const`?

Reply:
Refers to the `args` variable?

Reply:
No - contrary to what the name `args` suggests, it cannot be const because it contains `coins_to_uncache`, which is populated with coins and returned. If `coins_to_uncache` were removed from `ATMPArgs` then it could be const.

As a side thought, I personally think it would be nice/more RAII-like if uncaching would be handled by `MemPoolAccept` (see #21146), which is one approach to removing it from `args`.

Reply:
It seems this change also implies changing `MemPoolAccept::AcceptSingleTransaction`. Maybe it can be a follow-up PR.

Reply:
Just to clarify, I didn't mean to suggest that you change it - out of scope for this PR. Marking resolved.